jobs/JGDAS_ATMOS_VERFRAD

```diff
@@ -3,48 +3,60 @@
 #############################################################
 # Set up environment for GDAS Radiance Monitor job
 #############################################################
-source "$HOMEgfs/ush/preamble.sh"
+source "${HOMEgfs}/ush/preamble.sh"
+
+###########################################################
+# make temp directories
+###########################################################
+export DATA=${DATA:-${DATAROOT}/${jobid}}
+mkdir -p ${DATA}
+cd ${DATA}
+
+
+##############################################
+# Run setpdy and initialize PDY variables
+##############################################
+export cycle="t${cyc}z"
+setpdy.sh
+. ./PDY
+
+
+####################################
+# Determine Job Output Name on System
+####################################
+export pid=$$
+export pgmout="OUTPUT.${pid}"
+export pgmerr=errfile
+
 
 ###############################
 # Specify NET, RUN, and COMPONENT name
 ##############################
 export NET=${NET:-gfs}
 export RUN=${RUN:-gdas}
-export COMPONENT=${COMPONENT:-atmos}
+export COMPONENT="atmos"
 
-########################################################### 
-# obtain unique process id (pid) and make temp directories
-###########################################################
-export pid=$$
-export outid=${outid:-"LL$job"}
-export RAD_DATA_IN=${RAD_DATA_IN:-${DATAROOT}/${jobid:?}}
+export RAD_DATA_IN=${DATA}
 
 export RADMON_SUFFIX=${RADMON_SUFFIX:-${RUN}}
 export CYCLE_INTERVAL=${CYCLE_INTERVAL:-6}
 
-mkdir -p $RAD_DATA_IN
-cd $RAD_DATA_IN
-
-####################################
-# Determine Job Output Name on System
-####################################
-export pgmout="OUTPUT.${pid}"
-export pgmerr=errfile
-export cycle=t${cyc}z
+mkdir -p ${RAD_DATA_IN}
+cd ${RAD_DATA_IN}
```

Comment:
Can we just use `${DATA}` instead of `${RAD_DATA_IN}`?

Reply:
These lines are technically not necessary, as we are already in `DATA` from lines 12, 13.